.github/dependabot.yml

```diff
@@ -9,6 +9,7 @@ updates:
         patterns:
           - "*maven*plugin*"
           - "org.apache.maven*:*"
+          - "io.fabric8:docker-maven-plugin"
```

Comment:
Weird... shouldn't the first pattern have caught this plugin already? :thinking:

Reply:
hmm, right ... let me test it on my fork. If it did and the PR docker plugin PR was fine this morning, I could only explain it by a temporary blindness or something because I was looking for the maven plugin group on that PR and wasn't seeing it 😕 🫤 😖

Reply:
ok confirmed, it does work without this additional pattern in the list https://github.com/marko-bekhta/search.quarkus.io/pull/28
not sure why I didn't see the `maven-plugins group` or if it was or wasn't there .... 😖 
I'll remove the pattern in the dependabot PR so that I don't generate even more noise commits/merges with this change 😔